script/Arcueid/Day 6/06_00_ARC06_2_1.txt

```diff
@@ -130,13 +130,13 @@ Though I'm glad that it's only a fracture.
 [sha:1eb266d53a41d004962503e2b7faf049eff9be34]{
 -- Page 184, Offset 14358.
 -- 所々、折れていても不思議ではなかったんだから。
-I honestly wouldn't be surprised if I had quite a few fractures here and there.
+I honestly wouldn't be surprised if things were broken all over my body.
```

Comment:
Might lean towards keeping consistent with fracture

Reply:
other choice scene had it different so I took it from there.